ui/src/timeMachine/actions/index.ts

```diff
@@ -603,6 +604,18 @@ export const setHistogramPosition = (
   payload: {position},
 })
 
+interface SetLinePositionAction {
```

Comment:
we're slowly trying to phase this style out in favor of:
https://github.com/influxdata/influxdb/blob/master/ui/src/variables/actions/index.ts#L85

Reply:
@TCL735 if you convert this over, [here's a diff of a similar change i made recently.](https://github.com/influxdata/influxdb/commit/a90899ebf2e7908d154887bb05b20a22d5281945#diff-b0c7bc23d90cf32723c3bb8675444598R18) maybe it'll help you out with some of the issues i had. in particular, the [`ACTION_TYPE AS ACTION_TYPE`](https://github.com/influxdata/influxdb/commit/a90899ebf2e7908d154887bb05b20a22d5281945#diff-b0c7bc23d90cf32723c3bb8675444598R45) changes tripped me up.

Reply:
@hoorayimhelping thank you for code example, very helpful! I have refactored this into the preferred style.